vue/components/ui/link-platforme/link-platforme.vue

```diff
@@ -30,6 +30,15 @@ a.link.hover {
 a.link.link-small {
     font-size: 12px;
 }
+
+a.link.link-disabled {
+    color: #afafba;

```

Comment:
Use a named color. For now you can place it in `variables.css` but its name should match what you already have in the other branch.